workers/unity/Packages/com.improbable.gdk.core/Worker/ViewDiff.cs

```diff
@@ -0,0 +1,307 @@
+                    }
+                }
+            }
+        }
+
+        // public void MarkAuthorityTemporarilyLost(long entityId, uint componentId)
+        // {
+        //     if (!componentIdToComponentStorage.TryGetValue(componentId, out var authorityStorage))
+        //     {
+        //         throw new ArgumentException("Unknown component");
+        //     }
+        //
+        //     ((IDiffAuthorityStorage) authorityStorage).AddAuthorityChange(
+        //         new AuthorityChangeReceived(authority, new EntityId(entityId)));
+        //
+        //     authorityLostTemporarily.Add(new EntityComponent(entityId, componentId));
+        // }
+
+        public void AddComponentUpdate<T>(T update, long entityId, uint componentId, uint updateId)
+            where T : ISpatialComponentUpdate
+        {
+            if (!componentIdToComponentStorage.TryGetValue(componentId, out var storage))
+            {
+                throw new ArgumentException("Unknown component");
+            }
+
+            ((IDiffUpdateStorage<T>) storage).AddUpdate(new ComponentUpdateReceived<T>(update, new EntityId(entityId),
+                updateId));
+        }
+
+        public void AddEvent<T>(T ev, long entityId, uint componentId, uint updateId) where T : IEvent
+        {
+            if (!componentIdToComponentStorage.TryGetValue(componentId, out var storage))
+            {
+                throw new ArgumentException("Unknown component");
+            }
+
+            ((IDiffEventStorage<T>) storage).AddEvent(new ComponentEventReceived<T>(ev, new EntityId(entityId),
+                updateId));
+        }
+
+        public void AddCommandRequest<T>(T request, uint componentId, uint commandId) where T : IReceivedCommandRequest
+        {
+            var storage = componentIdToCommandIdToStorage[componentId][commandId];
+
+            ((IDiffCommandRequestStorage<T>) storage).AddRequest(request);
+        }
+
+        public void AddCommandResponse<T>(T response, uint componentId, uint commandId) where T : IRawReceivedCommandResponse
+        {
+            var storage = componentIdToCommandIdToStorage[componentId][commandId];
+
+            ((IDiffCommandResponseStorage<T>) storage).AddResponse(response);
+        }
+
+        public void AddCreateEntityResponse(CreateEntityResponseOp response)
+        {
+            worldCommandStorage.AddResponse(response);
+        }
+
+        public void AddDeleteEntityResponse(DeleteEntityResponseOp response)
+        {
+            worldCommandStorage.AddResponse(response);
+        }
+
+        public void AddReserveEntityIdsResponse(ReserveEntityIdsResponseOp response)
+        {
+            worldCommandStorage.AddResponse(response);
+        }
+
+        public void AddEntityQueryResponse(EntityQueryResponseOp response)
+        {
+            worldCommandStorage.AddResponse(response);
+        }
+
+        public void SetFlag(string name, string value)
```

Comment:
What are these empty public functions for?

Reply:
To get the API in place. These will need to work eventually.

Reply:
would prefer to either not have them in or add a todo with the ticket number. just having them completely empty like this is confusing.

Reply:
Removed them for now by popular demand.